### DISCUSSION TITLE
Does a readonly user exist?

### DISCUSSION THREAD

**Discussion by benisai** [lldap/lldap#113] [Q&A] [answered]
Hey, I got this to work and was wondering if a readonly user exists with limited rights. 

Thanks for the great work! 👍🏻

**Comment by nitnelave** [comment]
Hey,

Do you need a read-only user for the web interface, or for the LDAP
protocol?
The LDAP interface is already read-only, except for changing passwords.

**Reply by benisai** [reply]
LDAP protocol, what is the pw for the user: read-only?

**Comment by nitnelave** [comment]
There is no user able to write through the LDAP protocol (apart from the
passwords). Write operations are not supported. The admin user *is* the
"read-only" user.

**Reply by idlecoder-fr** [reply]
Hi,
First, thanks for the great job, this project is indeed exactly what I was looking for !

I am also looking for a "**read-only**" user : if one of my service using lldap for auth. is compromised, the admin creds will leak and thus could be used to connect to the "web admin" part.
Since the admin account can already be used through LDAP to list all known users, the problem is less the data leak than the risk to add a rogue user via the web admin part. Or have I missed something ?

Regards.

**Reply by nitnelave** [reply]
Well, you can always hide the web interface from the internet and only expose LDAP, but that's not ideal, I guess.

Can you create an issue? I'll probably add a group that allows reading through LDAP but no modification.

**Comment by nitnelave** [comment, marked as answer]
For the record, the readonly user has been implemented. When starting, LLDAP will create a `lldap_readonly` group to which you can add users. Users in this group can read all the information on the server but cannot modify anything except their own user info.

**Reply by benisai** [reply]
God bless you, for real.

**Comment by benisai** [comment]
I will give it a try tonight
